src/rt/aaA.d

```diff
@@ -477,7 +480,7 @@ extern (C) void _aaClear(AA aa) pure nothrow
 /// Rehash AA
 extern (C) void* _aaRehash(AA* paa, in TypeInfo keyti) pure nothrow
 {
-    if (!paa.empty)
+    if (!(*paa).empty)
```

Comment:
If you do keep `empty` as a non-member, I'd rather see it take a `const(AA*)` instead of this rewrite.